src/System.Private.ServiceModel/tests/Scenarios/Contract/Service/ServiceKnownTypeTests.cs

```diff
@@ -65,6 +65,20 @@ public static void ServiceKnownType_XmlSerializerFormat_AttrOnType_Test()
         RunTestMethodAndCleanup(factory, serviceProxy, serviceProxy.EchoItems_Xml, new Widget3());
     }
 
+    [Fact]
+    [OuterLoop]
+    public static void ServiceKnownType_XmlSerializerFormat_TwoOperationsShareKnownTypes_Test()
+    {
+        // *** SETUP *** \\
+        ChannelFactory<IServiceKnownTypeTest_AttrOnType_Xml> factory = GetChannelFactory<IServiceKnownTypeTest_AttrOnType_Xml>();
+        IServiceKnownTypeTest_AttrOnType_Xml serviceProxy = factory.CreateChannel();
+
+        // *** EXECUTE *** \\
+        // *** VALIDATE *** \\
+        // *** CLEANUP *** \\
+        RunTestMethodAndCleanup(factory, serviceProxy, serviceProxy.EchoItems_Xml1, new Widget3());
```

Comment:
Nit: don't block on this, but it would be very useful at some point to add comments to tests like this.  I've been staring at it for 5 minutes and still don't know what it actually tests or how it works.  Just something to consider when you need to touch this file again.